chain/actors/builtin/miner/diff.go

```diff
@@ -131,8 +145,8 @@ func DiffSectors(ctx context.Context, store adt.Store, pre, cur State) (*SectorC
 		return diffContainer.Results, nil
 	}
 	changes, err := diff.Amt(ctx, pres, curs, store, store, amt.UseTreeBitWidth(uint(preBw)))
-	if err != nil {
-		return nil, err
+	if span.IsRecording() {
```

Comment:
It looks like you've deleted the error checking code here

Reply:
oooffff I did in a rebase, great catch!